products/BellHybrid/apps/application-bell-main/ApplicationBellMain.cpp

```diff
@@ -72,6 +74,12 @@ namespace app
             requestShutdownWindow(gui::window::name::bell_battery_shutdown);
             return actionHandled();
         });
+
+        connect(typeid(sdesktop::usb::USBConfigured), [&](sys::Message *msg) -> sys::MessagePointer {
+            LOG_ERROR("Configured USB");
```

Comment:
are you sure there should be LOG_ERROR here?

Reply:
There should be no log :)
Good catch!

Reply:
Removed